update chainID to prevent replay attacks

lastwallfacer opened the pull request.

lastwallfacer committed:
update chainID to prevent replay attacks

ligi commented:
don't use 10001 - it is already used: https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-10001.json
so you will expose your chain to replay attacks still
also you will never be listed at e.g. chainlist.org
choose another one ..

mrtestyboy781 commented:
'PoW' in ascii is 50 6F 57 -> 5271383 in decimal

Make the chainId=5271383

rshi0614 commented:
anyone from the core ethw team doing this?

zquestz commented on params/config.go:
```diff
@@ -57,7 +57,7 @@ var CheckpointOracles = map[common.Hash]*CheckpointOracleConfig{
 	// MainnetChainConfig is the chain parameters to run a node on the main network.
 	MainnetChainConfig = &ChainConfig{
-		ChainID:             big.NewInt(1), //10001
+		ChainID:             big.NewInt(10001),
```
This chainID is currently used by smartBCH testnet.

I would suggest a different value. Perhaps 12001.

lastwallfacer replied:
八六九

to make a fortune, the fork will go smoothly and last forever

ate 69 for the lulz

lastwallfacer committed:
update chainID to unused and lucky ID

八六九

to make a fortune, the fork will go smoothly and last forever

ate 69 for the lulz
